modules/core/src/main/scala/com.snowplowanalytics.snowplow.bigquery/Environment.scala

```diff
@@ -55,7 +53,7 @@ object Environment {
       sourceReporter = sourceAndAck.isHealthy(config.main.monitoring.healthProbe.unhealthyLatency).map(_.showIfUnhealthy)
       appHealth <- Resource.eval(AppHealth.init[F, Alert, RuntimeService](List(sourceReporter)))
       resolver <- mkResolver[F](config.iglu)
-      httpClient <- BlazeClientBuilder[F].withExecutionContext(global.compute).resource
+      httpClient <- HttpClient.resource[F](HttpClient.Config(config.main.http.maxConnectionsPerServer))
```

Comment:
This can be more simply...

```scala
httpClient <- HttpClient.resource[F](config.main.http.client)
```

You don't need to re-construct the `HttpClient.Config` case class, because you already have one.